### DISCUSSION TITLE
how

### DISCUSSION THREAD

**Discussion by 2011pot** [ruffle-rs/ruffle#9614] [Q&A]
please tell me how to put flash onto google websites using html code and ruffle?

**Comment by coder0107git** [comment]
Are you meaning Google Sites? If you are, then use the embed button on the right pannel. Some of the  information for embedding Ruffle can be found [here](https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle#web). Any of the following sub-sections on that page should also help.
